app/views/brands/explore.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Explore eco-brands</h1>
+
+<h2>Brands</h2>
+<p>(waiting for seed)</p>
+<% @brands.each do |brand| %>
+    <%=brand.splash_image%>
+    <h2><%=brand.name%></h2>
+<% end %>
+
+<h2>News</h2>
+<p>(waiting for seed)</p>
+
+<%@newsarticles.each do |news|%>
+<%=news.image%>
+ <h2><%=news.title%></h2>
+ <p><%=news.blurb%></p>
```

Comment:
oh for articles do you also want to get the url, so that user can click on it and be taken to the external news article link @flaminiavg ?

Reply:
and new date too?